ggstat/agg.go

```diff
@@ -197,6 +197,15 @@ func AggSum(cols ...string) Aggregator {
 	return aggFn(vec.Sum, "sum ", cols...)
 }
 
+// AggPercentile returns an aggregate function that computes a
```

Comment:
Using the term "percentile" with a range [0, 1] was a mistake on my part. Let's not repeat it here. :) This should either be "quantile" with a range of [0, 1] or "percentile" with a range of [0, 100]. And either way the range should be documented.